### DISCUSSION TITLE
React SPA guide for this package

### DISCUSSION THREAD

**Discussion by rvibit** [Kyon147/laravel-shopify#83] [Show and tell]
I have migrated my app from BLADE to REACT and created a guide, Hope it'll be helpful for others https://softopia.in/how-to-create-a-shopify-app-with-laravel-shopify-package-and-react-spa/

@Kyon147 you can add it to the wiki if the community thinks it's useful.

**Comment by lucastes** [comment]
Thanks, I've been thinking about this lately and your post will be very helpful in making a decision

**Comment by ammar-yousaf** [comment]
> I have migrated my app from BLADE to REACT and created a guide, Hope it'll be helpful for others https://softopia.tech/how-to-create-a-shopify-app-with-laravel-shopify-package-and-react-spa/
> 
> @Kyon147 you can add it to the wiki if the community thinks it's useful.

This site url is not working. can you provide another guide for laravel shopify app with react?

**Reply by rvibit** [reply]
https://softopia.in/how-to-create-a-shopify-app-with-laravel-shopify-package-and-react-spa/
